### ISSUE TITLE
[New rule] Comment ignored lints

### ISSUE THREAD

**Issue by incendial** [dart-code-checker/dart-code-metrics#392]
**Please describe what the rule should do:**
The rule should recommend commenting `// ignore` in order to give more context why this ignore is applied.

**If your rule is inspired by other please provide link to it:**
Inspired by [flutter style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#comment-all-ignores).

**What category of rule is this? (place an "X" next to just one item)**

[ ] Warns about a potential error (problem)
[X] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about (it will be better if you can provide both good and bad examples):**

Bad:

```dart
// LINT
foo(); // ignore: lint_code
```

Good:

```dart
foo(); // ignore: lint_code, https://link.to.bug/goes/here

foo(); // ignore: lint_code, some text description
```

**Comment by incendial** [comment]
Available in 4.14.0 🚀
